lms/templates/dashboard/certificate_info.underscore

```diff
@@ -0,0 +1,111 @@
+<% if (model.may_certify && model.cert_status) {
+    if (model.cert_status.status === 'generating') {
+        status_css_class = 'course-status-certrendering';
+    } else if (model.cert_status.status === 'ready') {
+        status_css_class = 'course-status-certavailable';
+    } else if (model.cert_status.status === 'notpassing') {
+        status_css_class = 'course-status-certnotavailable';
+    } else {
+        status_css_class = 'course-status-processing';
+    }
+%>
```

Comment:
Please move this logic into the View so that a statusCssClass is passed into the template after being calculated in a getStatusClass function.